scripts/setupLift.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -e
+
+mkdir -p ~/.m2
+
+cat <<EOF > ~/.m2/settings.xml
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      https://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <localRepository>${user.home}/.m2/repository</localRepository>
```

Comment:
Setting `${user.home}` does not work. It provokes this failure
```

Setup's standard error:
/tmp/analyzing-bc07db667a225318/scripts/setupLift.sh: line 13: <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
                      https://maven.apache.org/xsd/settings-1.0.0.xsd">
  <localRepository>${user.home}/.m2/repository</localRepository>
  <interactiveMode>false</interactiveMode>
  <offline>false</offline>
</settings>
: bad substitution
```
That's why I had to force it to `/home/lift`

Reply:
Makes me worry that it's not the right path to use, but let's stick with it for now.

Maybe dump the env in the setup script to see what we can depend on

Reply:
I'm going to try the change suggested by @k-wall in my PR and if it works I will suggest here the same change.

Reply:
It is working fine adding using this approach:

```
M2_HOME=~/.m2

mkdir -p $M2_HOME
cat <<EOF > $M2_HOME/settings.xml
<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
                      https://maven.apache.org/xsd/settings-1.0.0.xsd">
  <localRepository>$M2_HOME/repository</localRepository>
  <interactiveMode>false</interactiveMode>
  <offline>false</offline>
</settings>
EOF
mvn install -DskipTests -pl :kroxylicious-krpc-plugin -Dquick
```